app/policies/episode_policy.rb

```diff
@@ -4,15 +4,24 @@ def create?
   end
 
   def update?
-    token&.authorized?(account_id, :episode) ||
-      ( token&.authorized?(account_id, :episode_draft) && resource.draft? && resource.was_draft? )
+    authorized?(:episode) || (authorized?(:episode_draft) && resource.draft? && resource.was_draft?)
   end
 
   def destroy?
-    token && token.authorized?(account_id, :admin)
+    authorized?(:admin)
   end
 
+  private
+
   def account_id
     resource.podcast.account_id
   end
+
+  def account_id_was
+    if resource.podcast_id_changed? && resource.podcast_id_was.present?
+      Podcast.find(resource.podcast_id_was)
```

Comment:
Ah, so also catching the case where they `set_podcast_uri`

Reply:
Yes - I actually think that account_id_was is unnecessary given the existing controllers but completeness doesn't hurt in this case.